### ISSUE TITLE
Call to Detections.__init__() missing 1 required positional argument: 'files'

### ISSUE THREAD

**Issue by yonomitt** [ultralytics/yolov5#5944]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Detection

### Bug

Calling `.tolist()` on a `Detections` object crashes with the following message:

```
  File "/****/.cache/torch/hub/ultralytics_yolov5_master/models/common.py", line 615, in <listcomp>
    x = [Detections([self.imgs[i]], [self.pred[i]], names=self.names, shape=self.s) for i in range(self.n)]
TypeError: __init__() missing 1 required positional argument: 'files'
```

There was a recent change (PR #5907), which fixed another bug. Unfortunately, it seems to have introduced this one.

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by yonomitt** [comment]
Don't worry, bot, I'm working on the PR.

**Comment by glenn-jocher** [comment]
@yonomitt good news 😃! Your original issue may now be fixed ✅ in PR #5945. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
